src/ovl_61_OpeningScene/29B410.c

```diff
@@ -129,13 +136,25 @@ INCLUDE_ASM(s32, "ovl_61_OpeningScene/29B410", func_800FBB94_2A09C4);
 
 INCLUDE_ASM(s32, "ovl_61_OpeningScene/29B410", func_800FBC9C_2A0ACC);
 
-INCLUDE_ASM(s32, "ovl_61_OpeningScene/29B410", func_800FBCC0_2A0AF0);
+Object* func_800FBCC0_2A0AF0(s32 arg0, void* arg1) {
+    Object* temp_v0;
+
+    temp_v0 = CreateObject(arg0 & 0xFF, arg1);
+    func_80025F60(*temp_v0->unk_3C->unk_40, 0x800);
+    func_80025F60(*temp_v0->unk_40->unk_40, 0x400);
+    return temp_v0;
+}
+
 
 INCLUDE_ASM(s32, "ovl_61_OpeningScene/29B410", func_800FBD14_2A0B44);
 
 INCLUDE_ASM(s32, "ovl_61_OpeningScene/29B410", func_800FBD48_2A0B78);
 
-INCLUDE_ASM(s32, "ovl_61_OpeningScene/29B410", func_800FBD7C_2A0BAC);
+s32 func_8004E184();                                  /* extern */
```

Comment:
If a function doesn't take arguments, be sure to define the signature as `s32 func_8004E184(void);`
leaving `()` blank means it takes an infinite amount of args